drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_assessment_loader.rb

```diff
@@ -113,14 +113,22 @@ def create_assessment_records(rows)
           log_skipped_row(row, field: ENROLLMENT_ID_COL)
           next # early return
         end
+
+        assessment_hud_key = row_assessment_id(row)
+        if @existing_assessment_hud_keys.include?(assessment_hud_key)
```

Comment:
I might be missing something but I think this instance variable will not be defined at this point; I don't see where we are calling the `existing_assessment_hud_keys()` function that would build the instance variable. 

It seems like the intention of `@existing_assessment_hud_keys` is to memoize the value of `existing_assessment_hud_keys()` - in that case I would suggest not using the instance variable directly (the ivar seems like more of a side-effect/function internal). Instead suggest always calling the `existing_assessment_hud_keys()` function

```suggestion
        if existing_assessment_hud_keys.include?(assessment_hud_key)
```

Reply:
whoops, totally my intention to use the method name, thanks!